pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -304,7 +304,7 @@ def init_ddp_connection(self, global_rank: Optional[int] = None, world_size: Opt
         world_size = world_size if world_size is not None else self.cluster_environment.world_size()
         os.environ["MASTER_ADDR"] = self.cluster_environment.master_address()
         os.environ["MASTER_PORT"] = str(self.cluster_environment.master_port())
-        if not torch.distributed.is_initialized():
+        if not torch_distrib.is_initialized():
```

Comment:
can we just keep the standard name? it seems so arbitrary to save 3 characters

Reply:
The rest of the file uses this abbreviation so I was just being consistent.

Can update all the occurrences

Reply:
aha ok, then whatever you feel is better.

Reply:
Done in 7a123543ab23c70920f9528a413479690a5d898e